crates/core/models/src/v0/channels.rs

```diff
@@ -306,4 +306,14 @@ impl Channel {
             | Channel::VoiceChannel { id, .. } => id,
         }
     }
+
+    pub fn name(&self) -> &str {
+        match self {
+            Channel::DirectMessage { .. } => "DMs",
```

Comment:
if this is used for generating notif, shouldn't this be using the recipient's name?

Reply:
It's not possible to fetch the name there, since it's a sync function and that would be a database call. Not really sure what else to do without refactoring everything to be async.

Reply:
Where is name used? Is it just the one occurrence? Are there many sync cases?

Reply:
Actually in the case of a DM for example, wouldn't the recipient already be known? (they can only be the message author - we don't see our own notifications)

Reply:
This is used in pushd (consumers/outbound/ (apn.rs and fcm.rs)), where that information isn't available. it appears to only be used in pushd, so I can update it to be async.

Reply:
Update - I can't reference the db since it's in models. I'll have to figure something else out.